src/compressed_tensors/compressors/model_compressor.py

```diff
@@ -90,10 +90,12 @@ def from_pretrained(
         configs and load a ModelCompressor
 
         :param pretrained_model_name_or_path: path to model config on disk or HF hub
-        :return: compressor for the extracted configs
+        :return: compressor for the configs, or None if model is not compressed
         """
         config = AutoConfig.from_pretrained(pretrained_model_name_or_path, **kwargs)
-        compression_config = getattr(config, COMPRESSION_CONFIG_NAME, None)
+        compression_config = getattr(config, QUANTIZATION_CONFIG_NAME, None) or getattr(
```

Comment:
I don't think this change is required atm, given the `SparseAutoModel` pathway will load any models with the quantization_config using a separate pathway through `from_compression_config`

Reply:
See: https://github.com/neuralmagic/compressed-tensors/pull/172#issuecomment-2381952277

Isn't it the case that `SparseAutoModel` will need to parse q_configs if the latest transformers is not installed?

Reply:
Yes but we are only supporting the `q_config` key for the latest transformers/ct version